src/main/java/org/folio/circulation/domain/RequestQueueRepository.java

```diff
@@ -38,6 +39,13 @@ public static RequestQueueRepository using(Clients clients) {
       .thenApply(result -> result.map(loanAndRelatedRecords::withRequestQueue));
   }
 
+  public CompletableFuture<Result<RenewalContext>> get(
+    RenewalContext renewalContext) {
```

Comment:
Why does this parameter need to be on a separate line, does the definition exceed 80 characters?

```suggestion
  public CompletableFuture<Result<RenewalContext>> get(RenewalContext renewalContext) {
```

Reply:
Fixed.